src/promptflow/promptflow/_cli/data/package_tool/utils.py.jinja2

```diff
@@ -17,6 +16,9 @@ def list_package_tools(raise_error=False):
     """
     # This function is auto generated by pf CLI, please do not modify manually.
     tools = {}
+
+    cache_file_name = hashlib.sha256(Path(__file__).parent.parent.as_posix().encode()).hexdigest()
```

Comment:
how do we handle package version?

Reply:
cache file will be generated in the built package. No need to handle package version